fabric-events-interaction-v0/src/main/java/net/fabricmc/fabric/api/event/player/DropItemCallback.java

```diff
@@ -0,0 +1,34 @@
+package net.fabricmc.fabric.api.event.player;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ActionResult;
+import net.minecraft.world.World;
+
+/**
+ * Callback for dropping an item.
+ * Is hooked in before the item is dropped.
+ * <p>
+ * Upon return:
+ * - SUCCESS cancels further processing and drops the item.
+ * - PASS falls back to further processing.
+ * - FAIL cancels further processing and does not drop the item.
```

Comment:
The behavior does not match the implementation, SUCCESS and FAIL currently both have the effect of FAIL.

Reply:
^^^^

Reply:
Should be fixed now.